content/en/docs/contribute/style/content-guide.md

```diff
@@ -45,68 +45,68 @@ effort from project maintainers and tends to become outdated more quickly.
 Before adding content, ask yourself this:
 
 - Is the content about an active CNCF project OR a project in the kubernetes or kubernetes-sigs GitHub organizations?
-    - If yes, then:
-        - Does the project have its own documentation?
-            - if yes, link to the project's documention from the Kubernetes documentation 
-            - if no, add the content to the project's repository if possible and then link to it from the Kubernetes documentation
-    - If no, then:
-        - Stop!
-            - Adding content about vendor-specific products is not allowed
-            - Linking to vendor-specific documentation and websites is not allowed
+    1. If yes, then:
+        1. Does the project have its own documentation?
+            1. if yes, link to the project's documention from the Kubernetes documentation 
+            1. if no, add the content to the project's repository if possible and then link to it from the Kubernetes documentation 
```

Comment:
nit: line 51, extra space at end; items start with lower case: `If yes`

Reply:
This will be done in a separate PR.  I've reverted my content-guide.md changes.